### PR TITLE
add doc comments to itokencache that were missing

### PR DESCRIPTION
**Pull request by MarkZuber**

### CODE REVIEW

File: src/Microsoft.Identity.Client/ITokenCache.cs [hunk 1]
```diff
@@ -25,21 +25,32 @@ public interface ITokenCache
     {
 #if !ANDROID_BUILDTIME && !iOS_BUILDTIME
         /// <summary>
-        ///
+        /// Sets a delegate to be notified before any library method accesses the cache. This gives an option to the
+        /// delegate to deserialize a cache entry for the application and accounts specified in the <see cref="TokenCacheNotificationArgs"/>.
+        /// See https://aka.ms/msal-net-token-cache-serialization
         /// </summary>
-        /// <param name="beforeAccess"></param>
+        /// <param name="beforeAccess">Delegate set in order to handle the cache deserialiation</param>
```
Comment (jennyf19) [inline src/Microsoft.Identity.Client/ITokenCache.cs]: deserialization? sp.

File: src/Microsoft.Identity.Client/ITokenCache.cs [hunk 2]
```diff
@@ -25,21 +25,32 @@ public interface ITokenCache
...
         /// <summary>
-        ///
+        /// Sets a delegate to be notified after any library method accesses the cache. This gives an option to the
+        /// delegate to serialize a cache entry for the application and accounts specified in the <see cref="TokenCacheNotificationArgs"/>.
+        /// See https://aka.ms/msal-net-token-cache-serialization
         /// </summary>
-        /// <param name="afterAccess"></param>
+        /// <param name="afterAccess">Delegate set in order to handle the cache serialization in the case where the <see cref="TokenCache.HasStateChanged"/>
+        /// member of the cache is <c>true</c></param>
+        /// <remarks>In the case where the delegate is used to serialize the cache entierely (not just a row), it might
```
Comment (jennyf19) [inline src/Microsoft.Identity.Client/ITokenCache.cs]: entirely? sp.

File: src/Microsoft.Identity.Client/ITokenCache.cs [hunk 3]
```diff
@@ -25,21 +25,32 @@ public interface ITokenCache
...
         /// <summary>
-        ///
+        /// Sets a delegate to be notified before any library method accesses the cache. This gives an option to the
+        /// delegate to deserialize a cache entry for the application and accounts specified in the <see cref="TokenCacheNotificationArgs"/>.
+        /// See https://aka.ms/msal-net-token-cache-serialization
         /// </summary>
-        /// <param name="beforeAccess"></param>
+        /// <param name="beforeAccess">Delegate set in order to handle the cache deserialiation</param>
+        /// <remarks>In the case where the delegate is used to deserialize the cache, it might
+        /// want to call <see cref="Deserialize(byte[])"/></remarks>
```
Comment (jmprieur) [inline src/Microsoft.Identity.Client/ITokenCache.cs]: or DeserializeMsalV3 ? (not a deprecated method ?) :)

File: src/Microsoft.Identity.Client/ITokenCache.cs [hunk 4]
```diff
@@ -25,21 +25,32 @@ public interface ITokenCache
...
-        ///
+        /// Sets a delegate to be notified after any library method accesses the cache. This gives an option to the
+        /// delegate to serialize a cache entry for the application and accounts specified in the <see cref="TokenCacheNotificationArgs"/>.
+        /// See https://aka.ms/msal-net-token-cache-serialization
         /// </summary>
-        /// <param name="afterAccess"></param>
+        /// <param name="afterAccess">Delegate set in order to handle the cache serialization in the case where the <see cref="TokenCache.HasStateChanged"/>
+        /// member of the cache is <c>true</c></param>
+        /// <remarks>In the case where the delegate is used to serialize the cache entierely (not just a row), it might
+        /// want to call <see cref="Serialize()"/></remarks>
```
Comment (jmprieur) [inline src/Microsoft.Identity.Client/ITokenCache.cs]: or rarther SerializeMsalV3?

**Comment by MarkZuber** [comment]
Good comments, @jmprieur.  This is a copy of what's on TokenCache.cs (they weren't on the interface).  I'll fix these up and get them in tomorrow.  thx!

**Pull request merged**